.goreleaser.yml

```diff
@@ -87,6 +87,13 @@ dockers:
   dockerfile: 'docker/kanister-kubectl/Dockerfile'
   build_flag_templates:
   - "--build-arg=TOOLS_IMAGE=ghcr.io/kanisterio/kanister-tools:{{ .Tag }}"
+- ids:
+  - kando
+  image_templates:
+  - 'ghcr.io/kanisterio/redis-sidecar:{{ .Tag }}'
```

Comment:
This image is not being used as "sidecar" in any pod. Let's name the image as `redis-tools` just like we have `ghcr.io/kanisterio/couchbase-tools` and `ghcr.io/kanisterio/mssql-tools`, etc

Reply:
fixed